crates/anstream/Cargo.toml

```diff
@@ -34,7 +34,7 @@ anstyle = { version = "1.0.0", path = "../anstyle" }
 anstyle-parse = { version = "0.2.0", path = "../anstyle-parse" }
 colorchoice = { version = "1.0.0", path = "../colorchoice", optional = true }
 anstyle-query = { version = "1.0.0", path = "../anstyle-query", optional = true }
-is-terminal = { version = "0.4.4", optional = true }
+is-terminal = { version = "0.4.6", optional = true }
```

Comment:
>  I would like to ask you to consider an upgrade to at least is-terminal v0.4.6 in order to fix documentation problems of Clap v4.1.8 and later with Rust v1.69.0.

Note: changing this crate is not needed for changing your crate.  You can run `cargo update -p is-terminal` and get the fix

Reply:
(this does ensure *we* can build our own documentation when we next release)

Reply:
Thank you for your help!  Your hint solved the problem for me, so I withdrew this PR.  For documentation, I added some information how to solve the problem to a Clap discussion, see https://github.com/clap-rs/clap/discussions/4855.